VL.ImGui.Stride/src/VL.ImGui.Stride.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0-windows</TargetFramework>
+    <TargetFramework>net8.0-windows7.0</TargetFramework>
```

Comment:
Again this guy ;) Still not sure about it, we have many other projects using `net8.0-windows` which do not need it.

Reply:
Because the window out and in again bug, i.e. losing the input handling, I will also take another close look at it. I'm just relatively busy at the moment. But hopefully I'll get back to it tonight or in the next few days. I think it also needs another cleanup

Reply:
Also works without ... is probably a leftover